instrumentation/github.com/Shopify/sarama/otelsarama/example/go.mod

```diff
@@ -2,8 +2,6 @@ module go.opentelemetry.io/contrib/instrumentation/github.com/Shopify/sarama/ote
 
 go 1.14
 
-replace go.opentelemetry.io/contrib => ../../../../../..
```

Comment:
We likely want to keep these replace directives to the parent directory for the `contrib` package. When rolling out a release we will updated the tagged version below prior to it actually being released. Without these declarations it would fail to build if there are breaking changes for the release.